### PR TITLE
dev-ruby/travis: bump version to 1.8.13

### PR DESCRIPTION
**Pull request by ferki**
This PR bumps dev-ruby/travis to its latest version.

Please review and merge, or let me know if further changes are required.

Closes: https://bugs.gentoo.org/717094

### CODE REVIEW

**Comment by graaff** [comment]
Please note that in general there is no real need to provide bumps for dev-ruby/* packages. They are already tracked against upstream and generally updated shortly after. Usually the fact that they are not indicates an issue. In this case, for me, travis 1.8.13 does not pass its test suite. Specifically:

```
  1) Travis::CLI::Cancel travis cancel
     Failure/Error: $params['id'].should be == "4125095"
     
       expected: == "4125095"
            got:    nil
     # ./spec/cli/cancel_spec.rb:6:in `block (2 levels) in <top (required)>'

  2) Travis::CLI::Cancel travis cancel 6180.1
     Failure/Error: $params['id'].should be == "4125096"
     
       expected: == "4125096"
            got:    nil
     # ./spec/cli/cancel_spec.rb:12:in `block (2 levels) in <top (required)>'
```

I have not investigated these failures yet. They may be legitimate or they may be caused by specifics in how Gentoo runs tests.

File: dev-ruby/travis/travis-1.8.13.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (graaff) [inline dev-ruby/travis/travis-1.8.13.ebuild:4]: Please try to update to the latest EAPI when bumping a version

File: dev-ruby/travis/travis-1.8.13.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+USE_RUBY="ruby23 ruby24 ruby25"
```
Comment (graaff) [inline dev-ruby/travis/travis-1.8.13.ebuild:5]: Please update the ruby targets. ruby23 is obsolete and should be removed. ruby26 and ruby27 should be tried to see if the package is ready for them (as determined by dependencies, test suite and changelog).

File: dev-ruby/travis/travis-1.8.13.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+RUBY_FAKEGEM_RECIPE_TEST="rspec"
```
Comment (graaff) [inline dev-ruby/travis/travis-1.8.13.ebuild:7]: This is an obsolete version of rspec. Please check if the package is now ready for rspec3.

**Comment by graaff** [comment]
Please note that there is no real need to make the changes I mentioned above, because I already had them ready in my own testing area.

**Comment by ferki** [comment]
Since [travis-1.9.0 is out since 2020-04-27](https://github.com/travis-ci/travis.rb#190-april-27-2020), I close this PR as obsolete. Will update the related bug about the newer version.

> Please note that in general there is no real need to provide bumps for dev-ruby/* packages. They are already tracked against upstream and generally updated shortly after. Usually the fact that they are not indicates an issue.

This will save me some time next time I try to figure out things in dev-ruby/*. Good to know about it, and thanks!

> Please note that there is no real need to make the changes I mentioned above, because I already had them ready in my own testing area.

I couldn't find dev-ruby/travis in your overlay and GitHub. Perhaps that testing area accessible somewhere else? I'd be happy test and learn about upcoming changes from there.

**Comment by graaff** [comment]
That testing area is private, no need to publish ebuilds that are known to be broken. In general if packages have not been added in a week or two after being released there is usually some kind of issue with them that needs to be resolved.

**Pull request closed without merging**